test_runner/regress/test_tenant_size.py

```diff
@@ -9,28 +10,240 @@ def test_empty_tenant_size(neon_simple_env: NeonEnv):
     env = neon_simple_env
     (tenant_id, _) = env.neon_cli.create_tenant()
     http_client = env.pageserver.http_client()
-    size = http_client.tenant_size(tenant_id)
+    initial_size = http_client.tenant_size(tenant_id)
 
-    # we should never have zero, because there should be the initdb however
-    # this is questionable if we should have anything in this case, as the
-    # gc_cutoff is negative
-    assert (
-        size == 0
-    ), "initial implementation returns zero tenant_size before last_record_lsn is past gc_horizon"
+    # we should never have zero, because there should be the initdb "changes"
+    assert initial_size > 0, "initial implementation returns ~initdb tenant_size"
 
-    with env.postgres.create_start("main", tenant_id=tenant_id) as pg:
+    main_branch_name = "main"
+
+    with env.postgres.create_start(main_branch_name, tenant_id=tenant_id) as pg:
         with pg.cursor() as cur:
             cur.execute("SELECT 1")
             row = cur.fetchone()
             assert row is not None
             assert row[0] == 1
         size = http_client.tenant_size(tenant_id)
-        assert size == 0, "starting idle compute should not change the tenant size"
+        assert size == initial_size, "starting idle compute should not change the tenant size"
```

Comment:
Hmm, as soon as you start compute, it can start generating WAL. For example, autovacuum, or a checkpoint. Those shouldn't happen immediately after starting the node, so I guess this is fine. It's a nice test to have, if it's reliable enough.

Reply:
Good point.
I disabled autovacuum and checkpointer for the test and left the comment.
If it will flake we'll probably have to loosen the check.

Reply:
In case you notice this from your notifications, comment about flakyness: This test has been with us from the first pr and hasn't come up as flaky even in default pg15 PR. test with multiple branches in it's name is the flaky one around branching for yet unknown reason.